### ISSUE TITLE
TAGS contains outdated paths

### ISSUE THREAD

**Issue by cordawyn** [hron/ruby_on_etags#7]
I'm not sure whether the case described below is actually the cause, but it seems to be very likely.

So, I've been using roetags for some time, while working with ruby 1.9.3-p0. Then I migrated to ruby 1.9.3-p125 (using rvm). Although I rebuilt the TAGS file again after that ("roetags build"), it still contained references to "ruby-1.9.3-p0" and thus was unable to find the required files. I removed the cache at ~/.ruby_on_tags and rebuilt the TAGS yet again. It worked then.
Apparently caching should take the ruby version into account too and do not mix the gems coming with different ruby versions.

**Comment by hron** [comment]
Yes, you are right.  I am aware of this issue. I was thinking it is best to use full path as a "cache key", because result TAGS files have full path in any case.
